exporter/mezmoexporter/README.md

```diff
@@ -27,10 +27,11 @@ that adds `hostname` detection support.
 
 # Configuration options:
 
-- `ingest_url` (optional): Specifies the URL to send ingested logs to.  If not 
+- `ingest_url` (optional): Specifies the URL to send ingested logs to.  If not
 specified, will default to `https://logs.mezmo.com/otel/ingest/rest`.
 - `ingest_key` (required): Ingestion key used to send log data to Mezmo.  See
 [Ingestion Keys](https://docs.mezmo.com/docs/ingestion-key) for more details.
+- `compression` (optional): Enables compression of requests.
```

Comment:
It would be good to add a note here that it's disabled by default, as well as a list of valid values, or an example value that could show what is expected here.